rule-packs/mitre-attck-initial-access-attack-paths.json

```diff
@@ -0,0 +1,326 @@
+[
+    {
+      "name": "initial-access-content-injection-aws",
+      "description": "M1041 - Encrypt Sensitive Information - Where possible, ensure that online traffic is appropriately encrypted using SSL/TLS enforcement in AWS.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND * WITH _integrationType = 'aws' THAT USES Configuration WITH parameter.ssl != 1 OR parameter.rds.force_ssl != 1",
+          "version": "v1"
+        }
+      ],
+      "alertLevel": "HIGH"
+    },
+    {
+      "name": "initial-access-content-injection-azure",
+      "description": "M1041 - Encrypt Sensitive Information - Where possible, ensure that online traffic is appropriately encrypted using SSL/TLS enforcement in Azure.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND * WITH _integrationType = 'azure' AND sslEnforced != true OR sslEnforcement != 'Enabled'",
+          "version": "v1"
+        }
+      ],
+      "alertLevel": "HIGH"
+    },
+    {
+      "name": "initial-access-content-injection-gcp",
+      "description": "M1041 - Encrypt Sensitive Information - Where possible, ensure that online traffic is appropriately encrypted using SSL/TLS enforcement in GCP.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND * WITH _integrationType = 'google_cloud' AND sslEnforced != true OR sslEnforcement != 'Enabled'",
+          "version": "v1"
+        }
+      ],
+      "alertLevel": "HIGH"
+    },
+    {
+      "name": "initial-access-exploit-public-facing-application-exploit-protection-storage",
+      "description": "M1050 - Exploit Protection - Ensure that public facing instances aren't allowed access to non-public S3 buckets",
```

Comment:
For the descriptions, we should avoid contractions for consistency; should update "aren't" to "are not"

Reply:
Also, this one is missing a period at the end of the description